src/com/artos/framework/infra/Runner.java

```diff
@@ -115,6 +115,24 @@ public void run(String[] args)
 			// Store only Upper case group names to avoid case sensitiveness
 			runnerTestSuite.getTestGroupList().replaceAll(String::toUpperCase);
 
+			// Add a default group if user does not pass a group parameter
+			{
+			if(null==runnerTestSuite.getTestBeforeUnitGroupList() || runnerTestSuite.getTestBeforeUnitGroupList().isEmpty()) {
+			runnerTestSuite.setTestBeforeUnitGroupList(Lists.newArrayList("*"));	
+			}
+			// Store only Upper case group names to avoid case sensitiveness
+			runnerTestSuite.getTestBeforeUnitGroupList().replaceAll(String::toUpperCase);
+			}
+		
+			// Add a default group if user does not pass a group parameter
+			{
+			if(null==runnerTestSuite.getTestAfterUnitGroupList() || runnerTestSuite.getTestAfterUnitGroupList().isEmpty()) {
+			runnerTestSuite.setTestAfterUnitGroupList(Lists.newArrayList("*"));	
+			}
+			// Store only Upper case group names to avoid case sensitiveness
+			runnerTestSuite.getTestAfterUnitGroupList().replaceAll(String::toUpperCase);
+			}
+			
```

Comment:
@SwapnaSoni I do not think we need any of this line. You should be utilizing @Group annotation so you dont need to set any defaults